packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -1070,6 +1086,7 @@ void debugPrintStack({StackTrace stackTrace, String label, int maxFrames}) {
   if (label != null)
     debugPrint(label);
   stackTrace ??= StackTrace.current;
+  stackTrace = FlutterError.demangleStackTrace(stackTrace);
```

Comment:
We can be confident that `StackTrace.current` never needs demangling - this is a bit of a nitpick, but I think

```dart
if (stackTrace == null) {
  stackTrace = StackTrace.current;
else {
  stackTrace = FlutterError.demangleStackTrace(stackTrace);
}
```

Of course, if we're already in an error state, any time this might save is pretty inconsequential.